packages/wpcom-proxy-request/package.json

```diff
@@ -35,8 +36,9 @@
 	],
 	"scripts": {
 		"clean": "npx rimraf dist",
+		"prepare": "tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json",
```

Comment:
🤔 Do we really need both for dev? Could we get away with only the esmodule variant and build the commonJS variant only in `prepublishOnly`?

Reply:
See https://docs.npmjs.com/misc/scripts#deprecation-note for details

Reply:
If we know we're using the `module` build from the main tsconfig, we could get away with a single build.

For TypeScript compiled packages, the best dev solution may be references implemented in #39786.

I've been wondering if we should even be using TypeScript to compile down to ES5. Lately I've been leaning more towards using TypeScript to compile to an intermediate version with a second pass where babel compiles to es5/cjs and es5/esm. There's room for exploration and improvement.

Reply:
> 🤔 Do we really need both for dev? Could we get away with only the esmodule variant and build the commonJS variant only in `prepublishOnly`?

I started playing with the idea of esm only builds for dev here: https://github.com/Automattic/wp-calypso/pull/43157